docs/modules/3d-tiles/api-reference/tiles-3d-loader.md

```diff
@@ -56,16 +56,28 @@ const tilesetJson = await load(tilesetUrl, Tiles3DLoader);
 const tilesetJson = await load(tilesetUrl, Tiles3DLoader, {'3d-tiles': {isTileset: true}});
 
 const tileset3d = new Tileset3D(tilesetJson, {
+  throttleRequests: false,
```

Comment:
Added this because in the case where you want to just fully load all tiles in a given view to render a scene, it was exiting too early and thus the code was doing a busy looping waiting for nothing. I think the default true here makes sense for an interactive scene with DeckGL.

Reply:
This sounds like a bug in the RequestScheduler. 

throttleRequests option should just prevent too many parallel requests from being issued and choking the network connection. It should improve overall performance and interactivity, but it certainly shouldn't semantically alter how the program works.